### ISSUE TITLE
Propose combine "istio-ca-secret" and "cacerts"

### ISSUE THREAD

**Issue by hzxuzhonghu** [istio/istio#41507]
(This is used to request new product features, please visit <https://discuss.istio.io> for questions on using Istio)

**Describe the feature request**

These two secrets are doing similar thing, the diff is "istio-ca-secret" is selfsigned, while "cacerts" is plugin.  

**Describe alternatives you've considered**

**Affected product area (please put an X in all that apply)**

[ ] Ambient
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Affected features (please put an X in all that apply)**

[ ] Multi Cluster
[ ] Virtual Machine
[ ] Multi Control Plane

**Additional context**

**Comment by howardjohn** [comment]
The idea SGTM but we will need to closely think about backwards compatibility, race conditions, etc

**Comment by jaellio** [comment]
@hzxuzhonghu what is the motivation behind combining the cacerts and istio-ca-sercret? Is the main goal simplification?

**Comment by hzxuzhonghu** [comment]
Yes, and to eliminate the confusion that both secrets represent same meaning

**Comment by jaellio** [comment]
Sounds good. I am writing up a proposal outlining the current functionality of each secret, the proposed solution, and considerations for backwards compatibility.

**Comment by jaellio** [comment]
@hzxuzhonghu Thanks for your initial feedback on the proposal! Ready for more reviews https://docs.google.com/document/d/17L1g0LgmZP1vs-R1WJrwJvKKZwUUHIRgcDPXuTEH_qM/edit?usp=sharing

**Comment by jaellio** [comment]
Part 1: https://github.com/istio/istio/issues/45685
Part 2: 
Part 3:
Part 4:

**Comment by hzxuzhonghu** [comment]
@jaellio are you continuing working on it

**Comment by hzxuzhonghu** [comment]
not stale
